From f325b8e08cd134187f3220524154e2b059ed0935 Mon Sep 17 00:00:00 2001
From: Timothy E Baldwin <T.E.Baldwin99@members.leeds.ac.uk>
Date: Fri, 4 Oct 2013 11:00:59 +0100
Subject: [PATCH 06/35] Fix Dynamic Areas comments

---
 castle/RiscOS/Sources/Kernel/s/ChangeDyn | 8 +-------
 1 file changed, 1 insertion(+), 7 deletions(-)

diff --git a/castle/RiscOS/Sources/Kernel/s/ChangeDyn b/castle/RiscOS/Sources/Kernel/s/ChangeDyn
index 2ba892a..dcf4717 100644
--- a/castle/RiscOS/Sources/Kernel/s/ChangeDyn
+++ b/castle/RiscOS/Sources/Kernel/s/ChangeDyn
@@ -523,7 +523,7 @@ readdyn_returnR2bit     *       &80
         BL      CheckAreaNumber                 ; out: r10 -> node
   ]
         Pull    "r1,lr"
-        BCC     %FT05                           ; [not a new one, so use old code]
+        BCC     %FT05                           ; Area does not exist, so return error
 
         CMP     r0, #&80                        ; CS => load maxsize into R2
                                                 ; (do this either if bit 7 set, or area >=&100)
@@ -5139,12 +5139,6 @@ AreaName_FontArea
 ; * Old CDA routine may be retired, since screen is its last client
 ; * Has Tim completed the rest of this work?
 ;
-; Once these routines work, they should be grafted into appropriate
-; places in the kernel sources
-;
-; This source is not intended for stand-alone assembly: it should be
-; plumbed into the kernel source build
-;
 ; Version history - remove this once integrated with kernel sources
 ;
 ; Vsn  Date      Who  What
-- 
1.8.4.rc3

